client/webserver/site/src/js/basepage.js

```diff
@@ -5,6 +5,11 @@ export default class BasePage {
     this.notifiers[note.type](note)
   }
 
+  registrationStatusUpdated (dexUrl) {
+    if (!this.onRegistrationStatusUpdated) return
+    this.onRegistrationStatusUpdated(dexUrl)
+  }
```

Comment:
This is only used on the `MarketsPage`, so it might as well trigger with the `feepayment` notification. Order of operations is

1. `Application` receives `feepayment` notification and sets appropriate fields in the `app.user.exchanges[url]`. 
2. `MarketsPage` receives `feepayment` notification and `MarketsPage.onRegistrationStatusUpdated(note.dexurl)`

Reply:
I like that! I made the changes as you suggested :)